### ISSUE TITLE
Apache License fails linter

### ISSUE THREAD

**Issue by khusmann** [bgruening/conda_r_skeleton_helper#41]
I have a r project with an Apache 2.0 license.

1) When using this helper, the license automatically pulled in becomes: "Apache License 2.0", which gets pulled into the license: recipe field

2) conda-forge-linter complains:
a) "The recipe license should not include the word 'License'."
b) "License is not an SPDX identifier (or a custom LicenseRef) nor an SPDX license expression."

3) fixed by changing to "Apache-2.0" (spdx identifiers are here: https://spdx.org/licenses/

Maybe conda_r_skeleton_helper could have a find/replace to translate the package license in CRAN to SPDX? This will prob be necessary for other licenses too.

Thanks!

**Comment by bgruening** [comment]
Yes, I think this is a good idea, especially since conda-forge will migrate t SPDX only, soon.

**Comment by dbast** [comment]
Related to this https://github.com/conda-forge/r-base-feedstock/issues/110

**Comment by dbast** [comment]
Almost all CRAN supported licenses (except Apache 2.0) are afaik packaged via the R upstream tarball und thus are also part of the conda r-base package. The individual r package feedstocks refer to those r-base included licenses in their meta.yml.

From that perspective a patch submitted to upstream to include also the Apache 2.0 license text file could be the first step here.

After that, updating the license dict in the conda-build cran skeleton file conda_build/skeletons/cran.py could be a next step. And while doing so, the dict can also be updated to spdx.

That maybe this is on a bit long time line and it makes sense to already include Apache-2.0 license file in r-base before it comes via upstream tarball. 

Any other thoughts?

**Comment by jameslamb** [comment]
I'd like to second the idea of generating the appropriate SPDX identifier. I just had to change `BSD_3_clause` to `BSD-3-Clause` on https://github.com/conda-forge/staged-recipes/pull/11382#issue-409132034
